src/lib/component/util/index.js

```diff
@@ -24,3 +24,5 @@ export { default as Timer } from "./Timer";
 export { default as UnauthenticatedRoute } from "./UnauthenticatedRoute";
 export { default as UnmountObserver } from "./UnmountObserver";
 export { default as useBuildInfo } from "./useBuildInfo";
+export { default as useQuery } from "./useQuery";
```

Comment:
Something I'm unclear about, do we need to export the types as well?

Reply:
I see that I do export `UseQueryOptions` and `UseQueryResult` from `useQuery.tsx` but don't pass them through here. If `util/index.js` itself were TypeScript we could export the additional types the same as regular identifiers:
```ts
export {
  default as useQuery,
  useQueryOptions,
  UseQueryResult
} from "./useQuery";
```
_Unlike Flow,TypeScript doesn't have a separate `type` identifier for importing/exporting types._

In any case, exporting them isn't necessary to use the typed `useQuery`.